src/main/kotlin/org/wycliffeassociates/otter/common/domain/ProjectPageActions.kt

```diff
@@ -48,17 +56,41 @@ class ProjectPageActions(
     }
 
     fun createNewTake(chunk: Chunk, project: Collection, chapter: Collection): Single<Take> {
-        return getTakeCount(chunk)
-                .map { numOfTakes ->
+        return getHighestTakeNumber(chunk)
+                .map { highestNumber ->
                     // Create a file for this take
+                    val languageSlug = project.resourceContainer?.language?.slug ?: ""
+                    val rcSlug = project.resourceContainer?.identifier ?: ""
+                    val bookNumber = "%02d".format(
+                            // Handle book number offset
+                            if (project.sort > 39) project.sort + 1 else project.sort
+                    )
+                    val bookSlug = project.slug
+                    val chapterNumber = "%02d".format(chapter.titleKey.toInt())
+                    val verseNumber = if (chunk.start == chunk.end) {
+                        "%02d".format(chunk.start)
+                    } else {
+                        "%02d-%02d".format(chunk.start, chunk.end)
```

Comment:
okay yes, the %02 is exactly what I said, but really it's a little more complicated than that- the 2 is the largest number of digits for that category; so Psalms should be %03 for chapters due to there being 150 chapters; and verses need to be %03 for psalm 119 because it has over 100 verses.

Takes though is fine at %02, regex for it just needs to change from [0-9]{2} to \d anyway. Likely 10 or more takes is going to be common, more than 100 seems unlikely, and it doesn't make sense to go rename old files to keep this consistent every time you add more files.

Reply:
no, I would call the repo, the main purpose is for compatibility with existing software

I believe this is defined in a spec somewhere too, I thought it was the resource container spec, but I'm having trouble finding it